.github/CODEOWNERS

```diff
@@ -1,3 +1,11 @@
 * @carlosrodlop
 * @holywen
-* @ps-ssingh
\ No newline at end of file
+* @ps-ssingh
+
+# Center of Excellence team as owners for files that require documentation review.
+
+*.md @cloudbees/team-docs-coe
```

Comment:
@carlosrodlop - do you mind updating, so `team-docs-coe` has write access to the repo?

Reply:
I will open a separate PR for this later.

Reply:
I will do it @kellie-freeman !